.CONTRIBUTING/.CONTRIBUTING.md

```diff
@@ -22,6 +43,79 @@ This guide covers how to contribute to the documentation, including serving a lo
     - [Image Annotations](#image-annotations)
   - [Search Engine Optimization (SEO)](#search-engine-optimization-seo)
 
+## Contribution Workflow
+
+### For Documentation Updates (Non-Tutorials)
+
+Follow these steps when contributing documentation fixes, improvements, or new content:
+
+1. **Fork and Create Branch**
+   ```bash
+   git clone https://github.com/your-username/polkadot-docs.git
+   cd polkadot-docs
+   git checkout -b your-feature-branch
+   ```
+
+2. **Make Your Changes**
+   - Edit existing files or add new content
+   - Follow the [style guide](https://github.com/papermoonio/documentation-style-guide/blob/main/style-guide.md)
+   - Test your changes locally (see [Viewing Site Locally](#viewing-site-locally))
+
+3. **Generate Updated AI Documentation**
+   After making content changes, regenerate the AI-ready documentation files:
+   ```bash
+   python3 scripts/generate_llms_standard.py
+   ```
+   This updates the `llms.txt` and `llms-full.txt` files that power our AI-ready documentation.
+   
+   **Note**: This script needs to be run on every PR once all changes have been committed.
+
+4. **Create Pull Request**
+   - Push your branch and create a pull request
+   - Use the PR template to indicate your review preference
+   - Include a clear description of your changes
+
+5. **Review Process Choice**
+   In your pull request, please let us know:
+   - **Option A**: "I have time to handle formatting and style review feedback myself"
+   - **Option B**: "Please have the Polkadot docs team take over for formatting review to expedite the process"
+
+   **If choosing Option B**:
+   - If you enable "Allow edits from maintainers" when creating your PR (this option appears for PRs from forks), our team can perform changes directly on your branch
+   - If "Allow edits from maintainers" is not enabled, we will create the `follow-up/PR-[original_pr_number]` branch and go through the intermediary workflow described below
+
+### Hand-off Process (Internal Team Workflow)
+
+When a contributor chooses Option B but does not have "Allow edits from maintainers" enabled, or when a contributor (regardless of option chosen) becomes unresponsive during review:
+
+1. **Create Intermediary Branch**: Create a new branch named `follow-up/PR-[original_pr_number]`
+2. **Redirect Original PR**: Change the destination of the original PR to the intermediary branch
+3. **Complete Formatting Review**: Perform style and formatting reviews on the intermediary branch
+4. **Final Merge**: Merge the completed intermediary branch into master
+
+If the contributor selects Option B AND allows edits from maintainers then we can perform changes directly on the contributor's branch.
+
+## Generating AI-Ready Documentation
```

Comment:
This should be part of the contribution workflow on line 46. Consider using `###` for the heading and adjusting all sub-headings.